README.md

```diff
@@ -1,3 +1,10 @@
+## ⚠️ Deprecated!
+
+**This repository is deprecated.** <br />
+A more up-to-date version of examples is available here: https://github.com/grafana/grafana-plugin-examples
```

Comment:
Should we migrate this starter-datasource into the examples or point this to the basic-datasource?

Reply:
https://github.com/grafana/grafana-starter-panel/pull/39#discussion_r839561009

Reply:
I think I will update it to point to the `basic-datasource` for now.
(We can always change it in the future if we find a more clear solution).